### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by comictomcat**
Old translation was a bit outdated. I've added language selection at the very top and moved all the links to the bottom, just like the main README file. I made translation clearer, although, I doubt it'll make a huge difference, since only a few lines were changed.

### CODE REVIEW

File: README.ru.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
 # sway
 
-sway - это [i3](https://i3wm.org/)-совместимый композитор [Wayland](http://wayland.freedesktop.org/).
-Больше информации в [FAQ](https://github.com/swaywm/sway/wiki). Присоединяйтесь к
-[IRC-каналу](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway на
+**[English][en]** - [日本語][ja] - [Français][fr] - [Українська][uk] - [Español][es] - [Polski][pl] - [中文-简体][zh-CN] - [Deutsch][de] - [Nederlands][nl] - [Русский][ru] - [中文-繁體][zh-TW] - [Português][pt] - [Danish][dk] - [한국어][ko] - [Română][ro]
```
Comment (emersion) [inline README.ru.md:3]: Other translated pages don't have the list of all languages. Since it becomes pretty easily outdated, can we omit it here as well?

Comment (comictomcat) [reply README.ru.md]: Sure, I made a commit

**Comment by emersion** [comment]
cc @DarkReef @YaLTeR

File: README.ru.md [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
 
-sway доступен во многих дистрибутивах. Попробуйте установить пакет "sway".
-Если он вдруг недоступен, проверьте [эту страницу на wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages)
-для получения информации о подробностях установки для вашего
-дистрибутива.
+Sway доступен во многих дистрибутивах. Попробуйте установить пакет "sway".
 
-Если вы заинтересованы поддерживать sway в вашем дистрибутиве, загляните в наш IRC-канал
-или обратитесь на sir@cmpwn.com за советом.
+Если Вас интересует создание пакета sway для вашего дистрибутива, зайдите [IRC-канал][IRC channel]
```
Comment (YaLTeR) [inline README.ru.md:18]:
```suggestion
Если вас интересует создание пакета sway для вашего дистрибутива, зайдите на [IRC-канал][IRC channel]
```

Вы с большой — скорее такой канцелярит (и в других местах с маленькой). Ну и "на".

File: README.ru.md [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
+или отправьте письмо на sir@cmpwn.com за советом.
 
 ### Сборка из исходников
 
+Посетите [эту страницу на вики][Development setup], если вы хотите создать HEAD
```
Comment (YaLTeR) [inline README.ru.md:23]:
```suggestion
Посетите [эту страницу на вики][Development setup], если вы хотите если вы хотите построить последнюю версию
```

**Review by YaLTeR — CHANGES_REQUESTED** [review]

File: README.ru.md [hunk 4]
```diff
@@ -1,42 +1,41 @@
...
+Sway доступен во многих дистрибутивах. Попробуйте установить пакет "sway".
 
-Если вы заинтересованы поддерживать sway в вашем дистрибутиве, загляните в наш IRC-канал
-или обратитесь на sir@cmpwn.com за советом.
+Если вас интересует создание пакета sway для вашего дистрибутива, зайдите на [IRC-канал][IRC channel]
+или отправьте письмо на sir@cmpwn.com за советом.
 
 ### Сборка из исходников
 
+Посетите [эту страницу на вики][Development setup], если вы хотите если вы хотите построить последнюю версию
```
Comment (gmykhailiuta) [inline README.ru.md:23]: Shouldn't there be only one "если вы хотите"?

Comment (comictomcat) [reply README.ru.md:23]: Yep

**Comment by emersion** [comment]
@gmykhailiuta, @YaLTeR, thanks for the suggestions! Does this PR now look good to you?

File: README.ru.md [hunk 5]
```diff
@@ -1,42 +1,41 @@
...
-sway доступен во многих дистрибутивах. Попробуйте установить пакет "sway".
-Если он вдруг недоступен, проверьте [эту страницу на wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages)
-для получения информации о подробностях установки для вашего
-дистрибутива.
+Sway доступен во многих дистрибутивах. Попробуйте установить пакет "sway".
 
-Если вы заинтересованы поддерживать sway в вашем дистрибутиве, загляните в наш IRC-канал
-или обратитесь на sir@cmpwn.com за советом.
+Если вас интересует создание пакета sway для вашего дистрибутива, зайдите на [IRC-канал][IRC channel]
+или отправьте письмо на sir@cmpwn.com за советом.
```
Comment (gmykhailiuta) [inline README.ru.md:19]: Perhaps, "отправьте письмо на sir@cmpwn.com чтобы получить совет", or "обратитесь к sir@cmpwn.com за советом"?

**Comment by gmykhailiuta** [comment]
Besides another small thing - looks fine by me.

**Review by YaLTeR — APPROVED** [review]

**Review by gmykhailiuta — APPROVED** [review]

**Comment by emersion** [comment]
Thanks!

**Pull request merged**